content/docs/setup/kubernetes/platform-setup/minikube/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Minikube
 description: Instructions to setup Minikube for use with Istio.
-weight: 15
+weight: 1
```

Comment:
I don't think we should change the order from alphabetical to arbitrary.

Reply:
It's not arbitrary, it was has been there from the beginning. It appears the order got broken.